build-in-container/scripts/build-mariner.sh

```diff
@@ -67,14 +67,18 @@ while (( "$#" )); do
         echo "Error: Unsupported flag $1" >&2
         exit 1
         ;;
+        *) # unsupported argument
+        echo "Error: Unsupported argument $1" >&2
+        exit 1
+        ;;
     esac
 done
 
 source /mariner/scripts/setup.sh
 
 if [[ "${container_type}" == "build" ]]; then
     # exit if SPECS/ is empty
-    if [ ! "$(ls -A $SPECS_DIR)" ]; then exit; fi
+    if [ ! "$(ls -A $SPECS_DIR)" ]; then exit 1; fi
```

Comment:
Does somewhere else inform the user of the issue if they hit this exit 1 result, so they can fix it?

Reply:
If not, can you add a message for the user to help inform why they hit the exit 1?

Reply:
Yep, it's part of another PR, wherein I am adding an echo statement